meetings/2018-01-17/minutes.md

```diff
@@ -0,0 +1,417 @@
+# Scala LSP Meeting Minutes, 11 January 2018
+
+In attendance: Jon Pretty, Bill Venners, Jorge Vicente Cantero, P. Oscar
+Boykin, Dale Wijnand, Martin Odersky, Heather Miller, Ólafur Páll Geirsson
+(Olaf), Eugene Burmako, Nikolay Tropin, Stu Hood, Seth Tisue, Adriaan Moors,
+Eugene Yokata, Gabriele Petronella, Alexey Alekhin, Iulian Dragos, Vladimir
```

Comment:
Looks like Eugene's last name got misspelt here and in some other parts of the document.

Reply:
I just can't spell his name. Thanks; fixed now!